tb_plugin/torch_tb_profiler/profiler/gpu_metrics_parser.py

```diff
@@ -31,6 +32,9 @@ def __init__(self):
         self.occupancy_per_device = [[] for _ in range(consts.MAX_GPU_PER_NODE)]
         self.avg_occupancy_per_device = [None] * consts.MAX_GPU_PER_NODE
         self.occupancy_count = [0] * consts.MAX_GPU_PER_NODE
+        # For calculating Tensor Cores time ratio.
+        self.kernel_per_device = [[] for _ in range(consts.MAX_GPU_PER_NODE)]
+        self.tc_ratio = [None] * consts.MAX_GPU_PER_NODE
```

Comment:
do we still need keep kernel parser tc_used_ratio?

Reply:
Yes. tc_ratio is an array of every GPU ratio. tc_used_ratio is ratio of all GPUs' kernels.
tc_used_ratio is used to show pie chart in kernel view:
![image](https://user-images.githubusercontent.com/10429114/133760348-0e0ad470-fd42-4f61-92f1-44952e79e8f0.png)